app/views/nodes/items_table/_table.html.erb

```diff
@@ -2,9 +2,10 @@
   <div class="table-wrapper">
     <table class="table table-striped"
       data-behavior="dradis-datatable"
-      data-default-columns='["Title", "Created", "Updated"]'
-      data-item-name="<%= items.first.class.name.underscore %>"
-      data-local-storage-key="project.ce.<%= dom_id(current_project) %>.<%= items.first.class.name.underscore.pluralize %>_datatable">
+      data-default-columns=<%= @default_columns.fetch(class_name.to_sym, ['Title', 'Created', 'Updated', 'Validation']).to_json %>
+      data-item-name="<%= class_name %>"
```

Comment:
I do think `class_name` is better than dragging the `items.first...` list throughout, but perhaps, we can define it ourselves at the beginning of the partial via `<% class_name = items.first.class.name.underscore %> than have the caller pass botht he `items` collection, and also spell out the `class_name` for the collection's items?

Reply:
Fixed ✅